spec/requests/articles_request_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
Empty class here. Do we need it?

Reply:
While this is not in my other controller specs, I see from local history it has been present since the creation of the file and for some reason if I remove it, I get the following error:

```
NameError:
  uninitialized constant ArticlesController
# ./spec/controllers/articles_controller_spec.rb:1:in `<top (required)>'
```

Reply:
Hmm, that's interesting. When I pull your branch and remove the file it seems to work alright. Are you running the command as `rspec spec/controllers/articles_controller_spec.rb`?

Reply:
Yup!

```
megan.duffy_@macbook:~/WPE/OnboardingProject$ rspec spec/controllers/articles_controller_spec.rb

An error occurred while loading ./spec/controllers/articles_controller_spec.rb.
Failure/Error:
  RSpec.describe ArticlesController, type: :controller do
  
    describe 'GET #new' do
      context 'when user is logged in' do
  
        let(:user) { create(:user) }
  
        before(:each) do
          allow(controller).to receive(:current_user).and_return(user)
        end

NameError:
  uninitialized constant ArticlesController
# ./spec/controllers/articles_controller_spec.rb:3:in `<top (required)>'
No examples found.


Finished in 0.00004 seconds (files took 0.1953 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

We spoke earlier about how weird my IDE set up is right now though lol.

Reply:
This file should be deleted and if that causes errors on you IDE then it needs to be fixed within your development environment.

Reply:
It looks as though this is because rails_helper was not set to autoload in .rspec.